models/snowflake/channel_counts.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+    channel,
+    campaign,
+    source,
+    medium,
+    COUNT(*) AS number_of_events
```

Comment:
Although we are using page_view events, we aren't really thinking of them as page views, as they are just events with associated marketing parameters, either multiple events or one (depending on the value of consider_intrasession_channels). So this would more accurately be number_of_sessions, as each event is being treated as a session, in the GA sense of a session. (Also see comment below)